pkg/kubelet/dockershim/helpers.go

```diff
@@ -73,6 +73,8 @@ func makeLabels(labels, annotations map[string]string) map[string]string {
 	for k, v := range annotations {
 		// Assume there won't be conflict.
 		merged[fmt.Sprintf("%s%s", annotationPrefix, k)] = v
+		// Add annotation without prefix
```

Comment:
The referenced issue claims:

"This is useful for sidecar containers that look for certain container labels, but are unable to because of the additional prefix added."

Can you elaborate on the use case?  What is the side car container?  Is the container deployed in the same pod?  Why does the container depend on these labels versus normal intra-pod communication?

More detail on the actual use case would help so we can properly transition use cases for users that adopt containerd/cri-o or any other runtime.

Reply:
This would help with services like registrator or traefik where it is looking for certain docker labels, but because of the annotation prefix sidecar services are unable to pickup docker labels added to the containers.

Reply:
Probably not using the term sidecar correctly. More so services looking for docker labels

Reply:
So containers deployed on host outside of Kubernetes?  Not an actual “sidecar” container which typically means a peer Container in same pod?

Reply:
No all containers are deployed via kubernetes manifests

Reply:
But yes, not a container in the same pod.

Reply:
So if traefik or registrator ran as a daemonset on a node but using docker labels for service discovery purposes